src/helpers/aws_helper.py

```diff
@@ -1846,3 +1848,47 @@ def execute_commands_on_ec2_by_tags_and_wait(
 
     console_printer.print_info(f"Response from ssm {resp}")
     time.sleep(timeout)
+
+
+def purge_sqs_queue(queue_name, aws_region="eu-west-2"):
+    console_printer.print_info(f"Purging queue: {queue_name}")
+    service_name = "sqs"
+    client = boto3.client(service_name, region_name=aws_region)
+    queue_url = client.get_queue_url(QueueName=queue_name)["QueueUrl"]
+    client.purge_queue(QueueUrl=queue_url)
+
+
+def execute_linux_command(
+    instance_id, linux_command, username="root", aws_region="eu-west-2"
+):
+    service_name = "ssm"
+    cmd = f'sudo su -c "{linux_command}" -s /bin/sh {username}'
+
+    client = boto3.client(service_name, region_name=aws_region)
+    response = client.send_command(
+        InstanceIds=[instance_id],
+        DocumentName="AWS-RunShellScript",
+        Parameters={"commands": [cmd]},
+    )
+    time.sleep(10)
+
+    command_id = response["Command"]["CommandId"]
+    output = client.get_command_invocation(InstanceId=instance_id, CommandId=command_id)
+
+    return output
+
+
+def upload_file_to_s3(bucket, object_key, metadata, body, aws_region="eu-west-2"):
```

Comment:
Is there not a method that already does this? I'd be surprised if not..

Reply:
Thanks Steve. changes applied.